docs/Earthfile

```diff
@@ -1,16 +1,35 @@
 VERSION 0.7
 
-deps:
-    FROM python:3.11-bullseye
-    DO ../earthly/python+POETRY_SETUP
+# cspell: words runable
 
+# Copy all the source we need to build the docs
 src:
-    FROM +deps
+    FROM ../earthly/docs+mkdocs-material
 
-    COPY --dir src mkdocs.yml .
+    # Just copy the actual doc src here.
+    COPY --dir src includes macros overrides mkdocs.yml /docs
 
+    # Now copy into that any artifacts we pull from the builds.
+    COPY --dir ../+repo-docs/repo /docs/includes
+
+
+# Build the docs here.
 docs:
     FROM +src
 
-    RUN poetry run mkdocs build --strict
-    SAVE ARTIFACT site/*
\ No newline at end of file
+    DO ../earthly/docs+BUILD
+
+    RUN /scripts/build.sh
+
+    SAVE ARTIFACT /site/*
+
+# Make a locally runable container that can serve the docs.
+local:
+    # Build a self contained service to show built docs locally.
+    DO ../earthly/docs+PACKAGE
+
+    # Copy the static pages into the container
+    COPY +docs/ /usr/share/nginx/html
+
+    # This is a local only image, we do not publish it.
```

Comment:
Just FYI, we don't use `--publish` at all anymore, because the CI handles the publishing. The comment seems to suggest otherwise.

Reply:
I am simply trying to highlight that this package is NOT a deployment package, it's for local use only.  It's also why I used `local` and not one of the normal targets.